modules/common/affinity/affinity.go

```diff
@@ -54,3 +62,78 @@ func DistributePods(
 		},
 	}
 }
+
+// DistributePodsOverride - returns rule to ensure that two replicas of the same selector
+// should not run if possible on the same worker node
+func DistributePodsOverride(
```

Comment:
What do you think about naming this function "DistributePodsWithOverrides"? This feels more intuitive to my english speaker ear.

Reply:
+1, I'm going to update it